### PR TITLE
Adds R6 notes

### PR DESCRIPTION
**Pull request by elias-garcia**
This PR adds release notes for R6.

### CODE REVIEW

**Review by otto-mora — CHANGES_REQUESTED** [review]
Testing to see if this allows me to commit suggestions after

File: docs/release-notes/r6.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+2. Allow the selection of a custom IPFS gateway
+3. Added Proof Type and Circuit ID inputs
+4. Implemented v3 circuits and proof of uniqueness in the form
+5. Return API timestamps expressed in UTC
+6. Implement off-chain verifications
+7. Implement on-chain verifications
+
+## Mobile app (2.2.9) - [Android](https://play.google.com/store/apps/details?id=com.polygonid.wallet) - [iOS](https://apps.apple.com/us/app/polygon-id/id1629870183)
+ 
+1. Refresh credential service
```
Comment (otto-mora) [inline docs/release-notes/r6.md:54]:
```suggestion
1. Implemented refresh credential service
```

File: docs/release-notes/r6.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+2. DisplayType
+3. Revoke credential from push notification
```
Comment (otto-mora) [inline docs/release-notes/r6.md:56]:
```suggestion
3. Support push notification for credential revocation
```

File: docs/release-notes/r6.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+4. Iden3commRevocationStatusV1.0 credential status type support
+5. Circuit V3 (experimental)
+6. Minor improvements
+
+## [Mobile SDK (2.2.9)](https://github.com/0xPolygonID/polygonid-flutter-sdk/releases/tag/v2.2.9)
+
+1. Refresh credential service
+2. DisplayType
+3. Credential status update iden3message
+4. Circuit V3 support
```
Comment (otto-mora) [inline docs/release-notes/r6.md:66]:
```suggestion
4. Included setting to support V3 of circuits (which are currently in beta)
```

**Review by otto-mora — CHANGES_REQUESTED** [review]
Minor wording change suggestion

File: docs/release-notes/r6.md (hunk 3 above)
Comment (elias-garcia) [reply docs/release-notes/r6.md:66]: Not really as this is the SDK, I'll add this exact phrase to the Mobile App.

**Review by otto-mora — APPROVED** [review]
Looks good 🖖

**Pull request merged**